src/lxc/af_unix.c

```diff
@@ -55,7 +55,7 @@ int lxc_abstract_unix_open(const char *path, int type, int flags)
 
 	addr.sun_family = AF_UNIX;
 
-	len = strlen(&path[1]) + 1;
+	len = strlen(&path[1]);
 	if (len >= sizeof(addr.sun_path) - 1) {
```

Comment:
`sizeof(addr.sun_path)` will give you `108` so `len(bla) >= 108 - 1` is correct.